jetstream_pt/config.py

```diff
@@ -139,6 +139,18 @@
     "size of top k used when sampling next token",
 )
 
+flags.DEFINE_integer(
```

Comment:
what is the boolean flag to enable / disable pagedattention?

Reply:
`self.page_attention` variable in JetEngineEnvironment are the flag. It's calculated by: 
`
  self.page_attention = self._data.total_num_pages > 0
`